<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>airlineWidget</class>
 <widget class="QWidget" name="airlineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:transparent
</string>
  </property>
  <widget class="QLabel" name="fromCityLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#808080</string>
   </property>
   <property name="text">
    <string>天河机场</string>
   </property>
  </widget>
  <widget class="QLabel" name="toCityLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线 Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#808080</string>
   </property>
   <property name="text">
    <string>双流机场</string>
   </property>
  </widget>
  <widget class="QLabel" name="fromTimeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>04:25</string>
   </property>
  </widget>
  <widget class="QLabel" name="toTimeLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>07:55</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>15</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#ffa631</string>
   </property>
   <property name="text">
    <string>355</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>15</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>南航222</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>35</y>
     <width>20</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#ffa631</string>
   </property>
   <property name="text">
    <string>￥</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>240</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img_ico/right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
